content/implementations.md

```diff
@@ -0,0 +1,81 @@
+---
+doc-note-type: draft
+layout: standalone_resource
+title: "Accessibility Test Tools & Methodologies"
+permalink: /standards-guidelines/act/implementations/
+ref: /standards-guidelines/act/implementations/
+lang: en
+type_of_guidance: false
+footer: ""
+github:
+  repository: w3c/wcag-act-rules
+  path: content/implementations.md
+---
+
+{%- assign siteData = site.data.wcag-act-rules %}
+{%- assign implementations = siteData.act-implementations | sort: "name" %}
+
+{::nomarkdown} {% include box.html type="start" title="Summary" %} {:/}
+ACT rules provide a standard way to compare the coverage of accessibility test tools and methodologies. This pages shows how many rules different tools and methodologies have accurately implemented. Scores are based on public test data.
```

Comment:
I think this paragraph does not communicate well what these pages are about. Will provide a separate pull request with proposed changes. GitHub does not let screen reader users apply suggestions to multiple lines at the moment.